drivers/bus/acpi/main.c

```diff
@@ -311,6 +311,11 @@ ACPIDispatchDeviceControl(
 
               status = STATUS_PENDING;
               break;
+               
+           // FIXME: When CORE-14134 gets fixed, IOCTL_BATTERY_QUERY_TAG case should be removed
```

Comment:
You don't need this comment -- because this is exactly where it will get fixed, so the case wouldn't get removed ;)